k256/src/arithmetic/field.rs

```diff
@@ -542,6 +542,29 @@ mod tests {
         assert_eq!(four.sqrt().unwrap().normalize(), two.normalize());
     }
 
+    #[test]
+    #[should_panic(expected = "assertion failed: self.normalized")]
```

Comment:
Another potential issue is that test will only succeed (i.e. `is_odd` will only panic) when `debug_assertions` are enabled.

Perhaps the test, or at least the `should_panic`, should be gated on that?

```suggestion
    #[cfg_attr(debug_assertions, should_panic(expected = "assertion failed: self.normalized"))]
```